core/modules/utils/dom/dom.js

```diff
@@ -311,6 +311,48 @@ exports.getLocationPath = function() {
 	return window.location.toString().split("#")[0];
 };
 
+exports.copyEventProperties = function(event) {
+	var seen = new Set();
+  
+	function isDOMElement(value) {
+	  return value instanceof Node || value instanceof Window;
+	}
+  
+	function safeCopy(obj) {
```

Comment:
This looks generally useful.  Should it be promoted to a public function?

Reply:
All functions in this file are available as `$tw.utils.functionNameHere`.